tests/foreman/api/test_computeresource_libvirt.py

```diff
@@ -63,88 +50,86 @@ def test_positive_create_with_name(setup, name):
 
     :parametrized: yes
```

Comment:
Can you please remove parameterized from docstring as well.

Reply:
why? this is parameterized on valid_data_list

Reply:
It looks like you removed the parametrization? https://github.com/SatelliteQE/robottelo/pull/12208/files#diff-1f56a82b56713dcb0ecea9d3197b5c51632fe3ffb7b39cd032d5e6bad5a271b4L52

Reply:
I merged create with name and description into single test as it was getting parametrized on same values, and it exists here https://github.com/SatelliteQE/robottelo/pull/12208/files#diff-1f56a82b56713dcb0ecea9d3197b5c51632fe3ffb7b39cd032d5e6bad5a271b4R104

Reply:
ah, gotcha, removed parametrize from  `test_positive_crud_libvirt_cr`, but this comment was misleading and was mentioning `test_positive_create_with_name`